### DISCUSSION TITLE
Clarity Question

### DISCUSSION THREAD

**Discussion by pjdreger3** [music-assistant/support#1204] [Q&A]
In the repository there are 

- server
- home-assistant-addon
- frontend
- hass-music-assistant

I get the impression that the server and the front end could be deployed on a separate device from an RPI running HASS and that the add-on is the piece running on the HASS box that would integrate them and allow control , automation etc .  it's this piece the HA Addon that's available via HACS and  was deprecated by the recent HA updates.  Which leave the hass-music assistant as the new beta intending to replace the HA Addon. 

Did I get that right?

**Comment by marcelveldt** [comment]
Nope,

server: is the actual music assistant core/server project you need to run somewhere.
addon: the server wrapped as HA addon for convenience.
frontend: the server's own UI (webapp) but it being a different programming language has its own repo
hass-music-assistant: currently hosts the V1 music assistant but next week this will host the HA integration to the server project.

Please read all announcements. The repo's are not important.

https://github.com/music-assistant/hass-music-assistant/issues/1143
